packages/dd-trace/src/supported-configurations.json

```diff
@@ -0,0 +1,432 @@
+{
+  "supportedConfigurations": {
```

Comment:
Why isn't this an array if strings? As far as I can see, we're just using each element to verify the if they are present or not... or have I misunderstood something?

Reply:
But am I correct that it's not used currently for anything? Maybe an example of how it is intended to be used would be nice

Reply:
The versions are needed. They are not used here in the repository but in the backend to verify the correct support in each library. It would not work without that.

The version declares the behavior of the individual configuration. That could be identical in another platform, it could however also be another default value, different behavior or similar and then have another version.